flax/linen/module.py

```diff
@@ -1488,6 +1488,29 @@ def init(self,
            **kwargs) -> FrozenVariableDict:
     """Initializes a module method with variables and returns modified variables.
 
+    Example::
+
+      from jax import random
+      import jax.numpy as jnp
+      import flax.linen as nn
+
+      model = nn.Dense(features=5)
+      rng, key = random.split(random.PRNGKey(0))
+      x = random.normal(key, (10,))
+      params = model.init(key, x)
+
+    Alternatively, if named collections are used. Example::
```

Comment:
Suggestion:

- The first sentence may read like it's not finished. You could rephrase it to imply that this is a second/alternative example. For example"

```suggestion
    Example (with named collections)::
```